doc/lua_api.txt

```diff
@@ -7002,7 +7002,11 @@ Used by `minetest.register_node`.
         -- Only valid for "nodebox" drawtype with 'type = "leveled"'.
```

Comment:
please change the definition of leveled above here to `leveled = 0` (the default value)